skims.cfg

```diff
@@ -53,12 +53,11 @@
     options = --outputs TPileupReweighting.prw.root
     dest = SFU-LCG2_LOCALGROUPDISK,
 [mc12_hadhad_ztautau]
```

Comment:
@vincecroft: please create a lephad  and leplep specific blocks in skims.cfg and datasets.cfg.
We don't want to edit this file everytime we want to test a different channel
Also we should add a soft link to the directory toward the testinput area so everyone has the same path